src/audio/eq_iir.c

```diff
@@ -797,33 +793,33 @@ static int eq_iir_prepare(struct comp_dev *dev)
 	if (ret == COMP_STATE_ALREADY_SET)
 		return PPL_PATH_STOP;
 
-	/* EQ components will only ever have 1 source and 1 sink buffer */
+	/* Get EQ source and buffers. */
```

Comment:
This will still be true since the SRC is not a mixer or mux

Reply:
OK, seems like I messed up the updated comment text ("sink" missing) but I'll revert to old since you prefer it.